### PR TITLE
Clarify PDB maxUnavailable rounding logic

### PR DESCRIPTION
**Pull request by i8r**
Clarify the implications of rounding-up in combination with using a percent value and maxUnavailable.

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
@i8r Thanks for making the contribution. Could you please sign the CLA before your PR can be reviewed.

For steps to sign the CLA please refer: https://github.com/kubernetes/community/blob/master/CLA.md

**Review by Arhell — COMMENTED** [review]
/easycla

**Comment by i8r** [comment]
@Arhell @dipesh-rawat Thanks for your patience. I've signed the CLA.

File: content/en/docs/tasks/run-application/configure-pdb.md [hunk 1]
```diff
@@ -129,8 +131,10 @@ of the number of desired replicas are healthy.
 Example 3: With a `maxUnavailable` of 5, evictions are allowed as long as there are at most 5
 unhealthy replicas among the total number of desired replicas.
 
-Example 4: With a `maxUnavailable` of 30%, evictions are allowed as long as no more than 30%
-of the desired replicas are unhealthy.
+Example 4: With a `maxUnavailable` of 30%, evictions are allowed as long as the number of 
+unhealthy replicas does not exceed 30% of the total number of desired replica rounded up to 
+the nearest integer. If the total number of desired replicas is just one, the only one replica
```
Comment (sftim) [inline content/en/docs/tasks/run-application/configure-pdb.md:136]:
```suggestion
the nearest integer. If the total number of desired replicas is just one, that single replica
```
?

**Review by dipesh-rawat — APPROVED** [review]
/lgtm

**Comment by i8r** [comment]
@sftim I've applied your suggestion. Thank you very much. Could you have another look and approve the PR again?

**Comment by reylejano** [comment]
Thank you for the clarification
/lgtm
/approve

**Pull request merged**